### ISSUE TITLE
Netflix clone not responsive

### ISSUE THREAD

**Issue by AdityaGujrati** [CleverProgrammer/netflix_clone#4]

**Comment by PurushothamanSrikanth** [comment]
@Adityagurati I think the project isn't fully done yet. @CleverProgrammer Is it done?

**Comment by Jonak-Adipta-Kalita** [comment]
> @Adityagurati I think the project isn't fully done yet. @CleverProgrammer Is it done?

Yes its done but @CleverProgrammer didnt uploaded the whole resources
